### DISCUSSION TITLE
Lead acid battery

### DISCUSSION THREAD

**Discussion by LuppiL** [pybamm-team/liionpack#274] [Q&A]
Hi, 
I'm trying to simulate a lead acid battery with the following code:
```

R_busbar = 1.5e-3
R_connection = 1e-2
model = pybamm.lead_acid.Full()
model = lp.add_events_to_model(model)

netlist = lp.setup_circuit(
    Np=Np, Ns=Ns, Rb=R_busbar, Rc=R_connection
)
param = pybamm.ParameterValues("Sulzer2019")

solver = pybamm.CasadiSolver(mode="safe")
sim = pybamm.Simulation(
    model=model,
    parameter_values=param,
    solver=solver,
)

experiment = pybamm.Experiment(
    [
        (
            "Charge at 3.0 A for 100 s",
        )
    ],
    period="1 s",
)
SoC = 0.5
# Solve pack
output = lp.solve(
    netlist=netlist,
    parameter_values=param,
    experiment=experiment,
    initial_soc=SoC,
    sim_func=sim,
)
```

but I get the following error: "'Maximum concentration in negative electrode [mol.m-3]' not found. Best matches are ['Initial concentration in electrolyte [mol.m-3]', 'Maximum porosity of negative electrode', 'Maximum porosity of positive electrode']".

How could I do to solve a model without using the highlighted parameters for a lead acid battery?

**Comment by TomTranter** [comment]
This package is not tested at all for Lead acid models and will be refactored in the future to use PyBaMM directly for solving to should hopefully address this issue. Apologies for the late reply
